environment/scripts/workspace_cleanup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]
+  then
+    echo "Please provide workspaces to be removed."
+fi
+
+in_use_workspaces="$@"
+reserved_workspaces="default production preproduction development develop integration training production02"
+
+protected_workspaces="$in_use_workspaces $reserved_workspaces"
+all_workspaces=$(terraform workspace list|sed 's/*//g')
+
+unset TF_WORKSPACE
+export TF_VAR_OPG_DOCKER_TAG=""
+export TF_EXIT_CODE="0"
+
+for workspace in $all_workspaces
+do
+  case "$protected_workspaces" in
+    *$workspace*)
+      echo "protected workspace: $workspace"
+      ;;
+    *)
+      if [[ $workspace == *"prod"* ]] || [[ $workspace == *"dependabot"* ]]
```

Comment:
We have a dependabot workspace?

Reply:
I was supposed to have removed this. Good spot! Fixed now